Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: install dist update
+SHELL=/bin/bash
+DATETIME:=$(shell date -u +%Y%m%dT%H%M%SZ)
+
+
+help: ## Print this message
+	@awk 'BEGIN { FS = ":.*##"; print "Usage:  make <target>\n\nTargets:" } \
+/^[-_[:alpha:]]+:.?*##/ { printf "  %-15s%s\n", $$1, $$2 }' $(MAKEFILE_LIST)
+
+install:
```

Comment:
I was making the incorrect assumption that we'd chain the commands so that `make dist` would call `make install`. If `make install` is just for dev, which it is in this case, I agree it make more sense to include `--dev`. Will fix.